tracer/test/snapshots/WcfTests.__metadataSchemaVersion=v1_binding=WSHttpBinding_enableNewWcfInstrumentation=False_enableWcfObfuscation=False.verified.txt

```diff
@@ -31,7 +31,7 @@
     TraceId: Id_1,
     SpanId: Id_4,
     Name: http.server.request,
-    Resource: WcfSample/ICalculator/ServerSyncAdd,
+    Resource: WcfSample/ICalculator/ServerAsyncAdd,
```

Comment:
I don't see any exceptions in the snapshots when enableNewWcfInstrumentation is `false`. I assume that's correct (given you've only updated the new one AFAICT) but just checking

Reply:
Yup, that's the point of enableNewWcfInstrumentation 🙂